Example/ScanDemo 2/Pods/CML_SCAN/README.md

```diff
@@ -0,0 +1,112 @@
+# Description
+
+This pod simplifies the implementation of barcode scanner. It supports scanning of EAN-8, EAN-13, code38, code128, QRCode and DataMatrix barcode types.
+
+# How To Get Started
+
+- Clone [CML_SCAN](https://sources.clasp-infra.com/tfs/DefaultCollection/Mobile/CML/_git/CML_SCAN) and try out the included iOS example app written in Swift.
+- Check out the [documentation](https://sources.clasp-infra.com/tfs/DefaultCollection/Mobile/CML/_git/CML_SCAN?path=%2FREADME.md&version=GBmaster&_a=contents) for a comprehensive look at all of the methods available in CML_SCAN.
+
+# Communication
+
+- If you need help or like to ask a general question, write an email to sukhjeetsingh.s@solcen.in or subramanya.n@solcen.in.
+
+- If you found a bug, and can provide steps to reliably reproduce it, open an issue on [JIRA](https://jasmin.clasp-infra.com/jira/issues/?jql=project%20%3D%20CML).
+
+- If you have a feature request, open an issue on [JIRA](https://jasmin.clasp-infra.com/jira/issues/?jql=project%20%3D%20CML).
+
+# Installation with CocoaPods
+
+CocoaPods is a dependency manager for Objective-C and Swift, which automates and simplifies the process of using 3rd-party libraries like CML_SCAN in your projects. You can install it with the following command:
+```
+$ sudo gem install cocoapods
+```
+
+### Podfile
+
+To integrate CML_SCAN into your Xcode project using CocoaPods, specify it in your Podfile:
+
+```
+platform :ios, '8.0'
+target 'TargetName' do
+  use_frameworks!
+  pod 'CML_SCAN', git: "https://sources.clasp-infra.com/tfs/DefaultCollection/Mobile/CML/_git/CML_SCAN"
+
+end
+```
+
+Then, run the following command:
+```
+$ pod install
+```
+
+# Requirements
+
+- iOS 8.0+
+- Xcode 8.0+
+- Swift 3.0+

```

Comment:
This is not a + unless you have actually tested on Swift 4, and it works (and not just in Swift 3 compatibility mode).